docs/reference/pages/theory.rst

```diff
@@ -95,3 +95,26 @@ For going beyond the basics of model definition and interrelation, it might help
     #.  A response object is returned by ``serve()`` and Django responds to the requester.
 
 You can apply custom behavior to this process by overriding ``Page`` class methods such as ``route()`` and ``serve()`` in your own models. For examples, see :ref:`model_recipes`.
+
+
+.. _scheduled_publishing:
+
+Scheduled Publishing
+~~~~~~~~~~~~~~~~~~~~
+
+Page publishing can be scheduled through the *Go live date/time* feature in the *Settings* tab of the *Edit* page. This allows you to set set up initial page publishing or a page update in advance and it will only happen at the
+scheduled time.
+In order for pages to be published at the scheduled time you should set up the :ref:`publish_scheduled_pages` management command.
+
+The basic workflow is as follows.
+
+* Scheduling a revision for a page that is not currently live schedules that page to go live when the scheduled time comes.
+* Scheduling a revision for a page that is already live schedules that revision to update the page when the time comes.
```

Comment:
schedules that revision to update the page -> means that revision will be published